packages/wrangler/src/config/environment.ts

```diff
@@ -98,50 +101,55 @@ interface EnvironmentInheritable {
 	 * Defaults to false.
 	 * If left unset, files will be named using the pattern ${fileHash}-${basename},
 	 * for example, `34de60b44167af5c5a709e62a4e20c4f18c9e3b6-favicon.ico`.
+	 *
+	 * @inheritable
 	 */
 	preserve_file_names: boolean | undefined;
 
 	/**
-	 * The directory in which module rules should be evaluated when including additional files into a worker deployment.
-	 * This defaults to the directory containing the `main` entry point of the worker if not specified.
+	 * The directory in which module rules should be evaluated when including additional files into a Worker deployment.
+	 * This defaults to the directory containing the `main` entry point of the Worker if not specified.
 	 *
 	 * @inheritable
 	 */
 	base_dir: string | undefined;
 
 	/**
 	 * Whether we use <name>.<subdomain>.workers.dev to
-	 * test and deploy your worker.
+	 * test and deploy your Worker.
 	 *
+	 * // Carmen according to our tests the default is undefined
```

Comment:
The default isn't quite undefined: it's a bit weird. It's documented https://developers.cloudflare.com/workers/configuration/routing/routes/#routes-with-workersdev, but essentially the default is `true` unless you have `route` or `routes` defined, in which case the default is `false`.